packages/components/pie-modal/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
should we create another ticket to share this between components?

Reply:
Yes, the ticket will be created soon. 👍